007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,181 @@
+    \texttt{unboundedFavour} &= \begin{cases}
+        \texttt{originalFavour} + 1, \quad\hfill \texttt{judgement} &> 0.075\\
+        \texttt{originalFavour} + \max{\left(\frac{\texttt{judgement}}{2}, -3\right)}, \quad\hfill\texttt{judgement} &< -2
+    \end{cases}\\
+    \texttt{favour} &= \begin{cases}
+        0, \quad\hfill \texttt{unboundedFavour} &< 0\\
+        \texttt{unboundedFavour}, \quad 0 \leq \texttt{unboundedFavour} &\leq 10\\
+        10, \quad\hfill \texttt{unboundedFavour} &> 10
+    \end{cases}\\
+\end{align*}
+
+Measuring favour allows the agent to act accordingly to how other agents are behaving around it, while taking on a passive conformist approach to decision making. If the agent views others around it more favourably, then it eats less food and aims for a lower HP in hopes of improving collective utility. The agent is also more likely to propose treaties to, and accept treaties from, highly favoured agents.
+
+For agents which have low favour, the Team 5 agent maintains its passiveness and does not ``punish'' them by, for example, taking more food than is necessary. The effect of ``punishing'' an agent propagates to all floors below the agent and is therefore not deemed an appropriate course of action. Instead, the agent strives to behave positively if surrounding agents also demonstrate positive behaviour.
+
+It should also be noted that while the Team 5 agent computes favour for agents both above and below its floor, a positive favour value for agents above is helpful to that agent only if they are moved to a floor below the Team 5 agent following a reshuffle: this is because there is no way to reward the behaviour of an agent above you in the tower.
+
+\section{Treaties}\label{sec:team5-treaties}
+The Team 5 agent's treaty strategy consists of three main parts:
+\begin{enumerate}
+    \item Deciding whether to accept or reject proposed treaties
+    \item When to propose a treaty
+    \item What to propose as a treaty
+\end{enumerate}
+
+\subsection*{Responding to Proposed Treaties}\label{sec:team5-treaties-response}
+A proposed treaty is always rejected if:
+\begin{itemize}
+    \item It conflicts with any other active treaties. A treaty is considered to conflict with another treaty if they could ever cause the agent to incompatible actions (for example, an agent cannot leave more than five units of food and fewer than four units of food at the same time).
+    \item It is badly formed. For example, a treaty which requires the agent to leave more than 100\% of the food on the platform is badly formed.
+    \item \ToDo{Check this one} It requires agents to leave less than a certain amount of food on the platform, as this is never advantageous to the tower as a whole.
+    \item \ToDo{Check this one} It requires the agent to leave an exact amount of food on the platform, as this would result ina ll but one of the agents signed up to the treaty to take no food each day.
```

Comment:
```suggestion
    \item \ToDo{Check this one} It requires the agent to leave an exact amount of food on the platform, as this would result in all but one of the agents signed up to the treaty to take no food each day.
```